core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -132,15 +137,21 @@ public void completeCreateView() {
             .withQuery("trino", "select * from ns.tbl using X")
             .withProperty("prop1", "val1")
             .withProperty("prop2", "val2")
-            .withLocation("file://tmp/ns/view")
+            .withLocation("file:///tmp/ns/view")
```

Comment:
the file scheme changes are for catalogs that use HadoopFileIO

Reply:
yes this was invalid, but I think it's still better to not directly rely on hardcoded directories under `/tmp` as that might interfere with other tests (running in parallel).